examples/src/Card.re

```diff
@@ -1,113 +1,117 @@
+open Belt;
+
 module DropTargetSpec =
-  BsReactDnd.DropTarget.MakeSpec(
-    {
-      type dragItem = T.dragItem;
-      type dropItem = T.dropItem;
-      type props = {
-        .
-        "id": int,
-        "moveCard": [@bs.meth] ((int, int) => unit),
-      };
-    },
-  );
+  BsReactDnd.DropTarget.MakeSpec({
+    type dragItem = T.dragItem;
+    type dropItem = T.dropItem;
+    type props = {
+      .
+      "id": int,
+      "moveCard": [@bs.meth] ((int, int) => unit),
+      "dropCard": [@bs.meth] (int => unit),
+    };
+  });
 
 module DropTargetWrapper =
-  BsReactDnd.DropTarget.Make(
-    {
-      let itemType = T.itemType;
-      type spec = DropTargetSpec.t;
-      let spec: spec =
-        DropTargetSpec.make(
-          ~hover=
-            (props, monitor, _component) => {
-              let dragItem = monitor##getItem() |> Js.toOption;
-              let hoverId = props##id;
-              switch (dragItem) {
-              | Some(dragItem) when dragItem##id !== hoverId =>
-                /* Time to actually perform the action */
-                props##moveCard(dragItem##id, hoverId)
-              | Some(_)
-              | None => ()
-              };
-            },
-          (),
-        );
-      type collectedProps = {. "connectDropTarget": BsReactDnd.Utils.wrapper};
-      type collect =
-        (BsReactDnd.DropTarget.connector, DropTargetSpec.monitor) =>
-        collectedProps;
-      let collect: collect =
-        (connect, _) => {"connectDropTarget": connect##dropTarget()};
-    },
-  );
+  BsReactDnd.DropTarget.Make({
+    let itemType = T.itemType;
+    type spec = DropTargetSpec.t;
+    let spec: spec =
+      DropTargetSpec.make(
+        ~hover=
+          (props, monitor, _component) => {
+            let dragItem = monitor##getItem() |> Js.toOption;
+            let hoverId = props##id;
+            switch (dragItem) {
+            | Some(dragItem) when dragItem##id !== hoverId =>
+              /* Time to actually perform the action */
+              props##moveCard(dragItem##id, hoverId)
+            | Some(_)
+            | None => ()
+            };
+          },
+        ~drop=
```

Comment:
the order will be updated on `hover`, when you drop the item nothing really changes. 

Here is the example I followed
demo: http://react-dnd.github.io/react-dnd/examples/sortable/simple
source: https://github.com/react-dnd/react-dnd/blob/master/packages/documentation-examples/src/04%20Sortable/Simple/Card.tsx

they only handle `hover`